packages/normalizr/src/denormalize.ts

```diff
@@ -228,7 +228,17 @@ function withTrackedEntities(
   const wrappedUnvisit = (input: any, schema: any) => {
     const ret: [any, boolean, boolean] = originalUnvisit(input, schema);
     // pass over undefined in key
-    if (ret[0] && schema && isEntity(schema)) globalKey.push(ret[0]);
+    if (ret[0] && schema && isEntity(schema)) {
+      /* istanbul ignore else */
+      if (typeof ret[0] === 'object') {
```

Comment:
```suggestion
      if (typeof ret[0] === 'object' || typeof ret[0] === 'function') {
```
alternatively and more robustly, `if (Object(ret[0]) === ret[0])`.

In the future, however, Symbols may be accepted as WeakMap keys, so this check will get a bit more complex.

Reply:
ret[0] is some value from a fetch response. While we need to ensure it is acceptable to use as a WeakMap key, we don't have to accept every possible type. I don't see purpose to allowing users to return both function or Symbol as they aren't JSON serializable nor some sort of data buffer.

As long as this check has no false positives, it is sufficient for these purposes. False negatives like banning function are quite acceptable.